ipaserver/install/dogtaginstance.py

```diff
@@ -285,10 +349,17 @@ def http_proxy(self):
             CLONE='' if self.clone else '#',
             FQDN=self.fqdn,
         )
+        if self.ajp_secret:
+            sub_dict['DOGTAG_AJP_SECRET'] = "secret={}".format(self.ajp_secret)
```

Comment:
IIRC our templating library will fail with a KeyError because a template variable is not defined.

```
else:
    sub_dict['DOGTAG_AJP_SECRET'] = ""
```